tests/unit/sagemaker/jumpstart/utils.py

```diff
@@ -31,15 +32,25 @@
 
 
 def get_header_from_base_header(
-    region: str = None, model_id: str = None, version: str = None
+    _obj: JumpStartModelsCache = None,
+    region: str = None,
+    model_id: str = None,
+    semantic_version_str: str = None,
+    version: str = None,
 ) -> JumpStartModelHeader:
 
+    if version and semantic_version_str:
+        raise ValueError("Cannot specify both `version` and `semantic_version_str` fields.")
+
     if "pytorch" not in model_id and "tensorflow" not in model_id:
         raise KeyError("Bad model id")
 
+    if region is not None and region not in JUMPSTART_REGION_NAME_SET:
+        raise ValueError(f"Bad region name: {region}")
```

Comment:
Can we change this to be:  `ValueError(f"Region name {region} not supported. Please use one of the supported region's in {JUMPSTART_REGION_NAME_SET}")`